### DISCUSSION TITLE
Thoughts on improving the mender template

### DISCUSSION THREAD

**Discussion by Efrank700** [silitics/rugix#35] [Ideas]
Hi Siltics Team!

I've just gotten into using RugPi after running face-first into the 64bit issue with mender. I saw your comments in the forums and have gotten set up with Rugpi and it's fantastic.

That said, given that you're positioned to become a defacto solution for new RPi-based mender deployments, it'd be awesome if the mender template were beefed up a bit to show use-cases.

Adding a few things like how to configure a rollback, configuration and preparation for a new update in the mender UI, and similar might help convince people to use the tool and accelerate adoption.

I'm currently working on the rollback stuff, so I'd be happy to share once it's complete.

**Comment by koehlma** [comment]
Great to hear that Rugpi and its Mender integration work well for you! :)

Indeed, improving the template and the documentation around it would be awesome. At the moment, our priorities are still on the core of Rugpi, and not so much on the integration with Mender. However, we definitely appreciate PRs improving the Mender integration and its documentation. Generally, Rugpi aims to be agnostic with respect to device management solutions (like Mender), giving users a choice and avoiding vendor lock-in.
